MediaBrowser.Controller/Sorting/AlphanumComparator.cs

```diff
@@ -121,7 +121,8 @@ public static int CompareValues(string? s1, string? s2)
                         return result;
                     }
                 }
-            } while (pos1 < len1 && pos2 < len2);
+            }
+            while (pos1 < len1 && pos2 < len2);
```

Comment:
Dislike this, but not sure how to fix

Reply:
dotnet also keeps it on the same line
https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/System/HexConverter.cs#L140